File: js_project/client/src/epubfile/css/RuleCss.js
import DeclCss from './DeclCss'
// Represents a CSS rule: a selector followed by a declaration block
export default class RuleCss {
    selector
    nodes
    constructor(props) {
        this.selector = props && props.selector
        const declNodes = props && props.nodes
        if (Array.isArray(declNodes)) {
            this.nodes = Array.of()
            for (let node of declNodes) {
                this.nodes.push(new DeclCss(node))
            }
        }
    }
}

File: lib/src/main/java/com/chineseall/epubparser/lib/css/RuleCss.kt
package com.chineseall.epubparser.lib.css

/**
 * Represents a CSS rule: a selector followed by a declaration block
 */
class RuleCss {
    var selector: String? = null // 选择器
    var nodes: MutableList<DeclCss>? = null // 样式列表
}